File: TenMinsJava/day57.java
// Check if string contains 8 or 10 digits
package TenMinsJava;

import java.util.Scanner;

public class day57 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.err.println("Enter 8 or 10 digits");
        String s = scan.nextLine();
        if ((s.length() == 8 || s.length() == 10) && s.matches("\\d+")) {
            System.err.println("nice");
        }
        else {
            System.err.println("nah");
        }
    }
}

File: TenMinsPy/day57.py
# Check if string contains 8 or 10 digits

class day57:
    def __init__(self):
        s = input("enter 8 or 10 digit num\n")
        if s.isdigit() and len(s) in (8, 10):
            print('nice')
        else:
            print('nah')

day57()